generator/src/test/java/com/scottlogic/deg/generator/cucumber/Before.feature

```diff
@@ -2,13 +2,13 @@ Feature: User can specify that a temporal date is lower than, but not equal to,
 
 Background:
      Given the generation strategy is full
+       And there is a field foo
+       And foo is of type "temporal"
+       And foo is anything but null
 
 Scenario: User requires to create a temporal field with date (YYYY-MM-DD) values within a given month that are less than a specified date
-     Given there is a field foo
-       And foo is before 2018-10-10T00:00:00.000
+     Given foo is before 2018-10-10T00:00:00.000
        And foo is after 2018-10-01T00:00:00.000
-       And foo is anything but null
-       And foo is of type "temporal"
```

Comment:
How come temporal type is removed?

Reply:
Its defined in the `Background` for the feature, so `foo` is temporal in all scenarios